cmds/choose.go

```diff
@@ -65,7 +66,8 @@ func Choose() *cli.Command {
 			}
 			choice := strings.TrimSpace(cmdOutput.String())
 			// TODO: get choice from Session structs array
-			connect.Connect(choice, false, "")
+			config := config.ParseConfigFile()
+			connect.Connect(choice, false, "", &config)
```

Comment:
Do you like passing all of the config into these functions? I'm not sure how to break up the config file yet but we may end up passing more data than is needed for each function.

Reply:
I think there are two approaches here: pass exactly what we need, or pass a reference to the config.Config struct. That way, we have all the data that is needed (and then some), but we aren't needlessly creating copies of the config struct. I'm open to either, but I will say that it's maybe a little hard to know at this point what we'll need to pass through for future config values and the parameters could get a bit out of hand if they're not all in a single object that you can pass through.